system/Config/DotEnv.php

```diff
@@ -116,7 +116,8 @@ public function normaliseVariable(string $name, string $value = ''): array
         $value = trim($value);
 
         // Sanitize the name
-        $name = str_replace(['export', '\'', '"'], '', $name);
```

Comment:
What is the purpose of deleting these strings?

Reply:
I'm not entirely sure, but I think this was meant to make `'foo'` or `"foo"` to just `foo`